src/join.rs

```diff
@@ -120,6 +121,17 @@ impl<J: Join> JoinIter<J> {
     }
 }
 
+impl<J: Join> JoinIter<J> {
+    /// Allows getting joined values for specific entity.
+    pub fn get(&mut self, e: Entity) -> Option<J::Type> {
+        if self.keys.contains(e.id()) {
```

Comment:
Thinking about it again, maybe we should have a note saying you need to check if the entity is alive prior to doing this?

Reply:
Yes, that's something I missed but spotted in your code later @Aceeri :P Somehow I'm more picky on Gitter I guess ^^

@WaDelma I think it would be preferable to just use `Index` as parameter instead of `Entity`.

Reply:
Why would `Index` be preferred? Only thing that uses it is `UnprotectedStorage` which is not something normal library user would touch and all user facing API uses `Entity`.

So the problem with dead entities is that if the index is reused it would get that one instead of returning `None`? I would change it to check the liveliness then.

Reply:
> I would change it to check the liveliness then.

But that would require to pass `Entities`.

Reply:
`Entities` or `World` depending where it's being called.

I just don't know if it's good idea to leave this for the user to check.

Reply:
You can get the entities from the world.

If it's a good idea, not sure, but it's better than using an entity without upholding the imposed guarantee.